src/components/newsFeedV2/newsFeed.css

```diff
@@ -0,0 +1,91 @@
+@import '../app/variables.css';
+@import '../app/variablesV2.css';
+
+.header {
+  justify-content: space-between !important;
+}
+
+.newsFeedBox {
+  padding: 0;
+  height: auto;
+  font-family: var(--content-font);
+}
+
+.settingsButton {
+  display: flex;
+  align-items: center;
+  font-weight: 600;
+
+  & span {
+    cursor: pointer;
+    color: #2475b9;
+  }
+}
+
+.emptyNews {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  font-family: var(--content-font);
+  font-size: var(--paragraph-font-size-l);
+  padding-top: 16px;
+}
+
+.liskLogo {
+  opacity: 0.2;
+}
+
+.container {
+  min-height: 578px;
+  overflow: hidden;
+  height: 578px;
+
+  & > div {
+    height: 100%;
+  }
+
+  & .newsWrapper {
+    box-sizing: border-box;
+    padding: 0 20px;
+  }
+}
+
+.newsFeed {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  height: 866px;
+}
+
+@media (--medium-viewport) {
```

Comment:
I think `--medium-viewport` shouldn't have so many specific styles. But I cannot check it because currently, it doesn't show at all in the medium viewport.
I could find this design for medium viewport layout: https://zpl.io/brv0jX3 not sure if it's still valid, please check with @reyraa